lib/src/license_detection/license.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:pana/src/license_detection/tokenizer.dart';
+
+@sealed
+class License {
+  /// Name of the license, is empty in case of unknown license.
+  final String licenseName;
```

Comment:
Isn't this SPDX identifier?
```suggestion
  final String spdxIdentifier;
```

Reply:
hmm, I also see in `LicenseMatch` we have `name`.

Maybe we should use consistent terminology. Because in both places these are SPDX identifiers.

We could use `spdxIdentifier` or `identifier` or `licenseIdentifier` or `spdxId`.

Maybe it's simplest to just use `identifier`, because it makes sense that `License.identifier` is an _SPDX identifier_, similarly it makes sense that `LicenseMatch.identifier` is an _SPDX identifier_.